### PR TITLE
fix(azure): Fix "branchName" field for azure-helper.ts

### PR DESCRIPTION
**Pull request by zharinov**
Ref: #7205

### CODE REVIEW

File: lib/platform/azure/util.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      [PullRequestStatus.Abandoned]: PrState.Closed,
+      [PullRequestStatus.Completed]: PrState.Merged,
+    }[azurePr.status] || PrState.Open;
+
+  const sourceRefName = azurePr.sourceRefName;
+
+  const isConflicted = azurePr.mergeStatus === PullRequestAsyncStatus.Conflicts;
+
+  return {
+    ...azurePr,
```
Comment (zharinov) [inline lib/platform/azure/util.ts:71]: For now, I'm afraid to get rid of old fields and restrict the result, so the whole thing should not to be destructive

**Review by viceice — COMMENTED** [review]
Renaming files makre this pr difficult to review, as is declared as a fix?

**Comment by zharinov** [comment]
> Renaming files makre this pr difficult to review, as is declared as a fix?

Right, the actual fix is single-line. But I think we need to make these renames anyway in later PRs, before we introduce Renovate-created PR filtering. That's because we mock `getRenovatePRFormat` function.

File: lib/platform/azure/util.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    return branchPath;
+  }
+  return branchPath.substring(5, branchPath.length);
+}
+
+export function getRenovatePRFormat(azurePr: GitPullRequest): AzurePr {
+  const number = azurePr.pullRequestId;
+  const displayNumber = `Pull Request #${number}`;
+
+  const branchName = getBranchNameWithoutRefsheadsPrefix(azurePr.sourceRefName);
```
Comment (zharinov) [inline lib/platform/azure/util.ts:53]: Actual fix

**Comment by zharinov** [comment]
Can confirm my test run was okay

**Comment by viceice** [comment]
> > Renaming files makre this pr difficult to review, as is declared as a fix?
> 
> Right, the actual fix is single-line. But I think we need to make these renames anyway in later PRs, before we introduce Renovate-created PR filtering. That's because we have to avoid `getRenovatePRFormat` mocks in `index.spec.ts`.

OK, please next time please split fix and refactor, so we can publish fixes more faster. 🙃

File: lib/platform/azure/util.ts (hunk 2 above)
Comment (viceice) [reply lib/platform/azure/util.ts:53]: ok, is see. We should rename `branchName` to `sourceBranch` to be more clean (other pr of cause).

File: lib/platform/azure/util.ts (hunk 1 above)
Comment (viceice) [reply lib/platform/azure/util.ts:71]: yes, we should only copy data we need.

**Review by viceice — APPROVED** [review]

**Pull request merged**